pages/blog-post.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Blog Post</title>
+    <title>News Post</title>
```

Comment:
Looking everything perfect, this should be Blog Post instead of News Post?

Reply:
@benjaminskarding I am merging, looks everything very nice. Just remember to change the News Post to Blog Post if necessary :)